src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2056,13 +2136,19 @@
           short-summary: Space-separated list of locations to configure.
         - name: --resource-group -g
           short-summary: Name of resource group. Required when enabling new regions.
-          long-summary: When a previously disabled region is enabled to use Network Watcher, a
-            Network Watcher resource will be created in this resource group.
+          long-summary: >
+            When a previously disabled region is enabled to use Network Watcher, a
+                Network Watcher resource will be created in this resource group.
+    examples:
+        - name: Configure Network Watcher in a region
```

Comment:
End example text with a `.`. There may be instances of this I missed in review, please check for others.

Reply:
Fixed in my branch.